modules/light-clients/06-solomachine/light_client_module.go

```diff
@@ -164,7 +167,10 @@ func (l LightClientModule) VerifyNonMembership(
 	return clientState.VerifyNonMembership(ctx, clientStore, l.cdc, height, delayTimePeriod, delayBlockPeriod, proof, path)
 }
 
-// Status obtains the client state associated with the client identifier and calls into the clientState.Status method.
+// Status returns the status of the solo machine client.
+// The client may be:
+// - Active: if frozen sequence is 0
+// - Frozen: otherwise solo machine is frozen
```

Comment:
I think this was a mistake in the original godoc for solomachine. Frozen sequence is used in tendermint clients, but solomachine clients only have the `IsFrozen` bool on the client state.

Reply:
```suggestion
// The client may be:
// - Active: if `IsFrozen` is false.
// - Frozen: if `IsFrozen` is true.
// - Unknown: if the client state associated with the provided client identifier is not found.
```